lib/vsc/install/shared_setup.py

```diff
@@ -159,7 +159,7 @@ def _log(self, level, msg, args):
 
 RELOAD_VSC_MODS = False
 
-VERSION = '0.14.0'
+VERSION = '0.14.2'
```

Comment:
why the jump? anything missing?

Reply:
See commit msg, I accidentally published a `0.14.1` to PyPI (with stuff not included in this PR). I removed that version again from PyPI shortly after (and you can't recycle version numbers on PyPI, rightfully so).